docs/configuration.md

```diff
@@ -120,6 +120,29 @@ jobs:
   trigger: commit
 ```
 
+**copr\_build**
+
+Gather data from new pull_request or release and use trigger new copr build with it.
+
+Supported triggers: **pull_request**, **release**.
+
+
+**Example**
+
+```yaml
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets:
+      - fedora-rawhide-x86_64
+      - fedora-30-x86_64
+    # (Optional) Defaults to 'packit'
+    owner: some_copr_project_owner
+    # (Optional) Defaults to <github_namespace>-<github_repo>
+    project: some_project_name
```

Comment:
the same, but it's awesome you made this configurable so we can set it to our repos when testing this